hbase-server/src/main/java/org/apache/hadoop/hbase/util/HBaseFsck.java

```diff
@@ -1929,7 +1929,7 @@ private void checkAndFixTableStates() throws IOException {
     }
   }
 
-  private void preCheckPermission() throws IOException, AccessDeniedException {
+  private void preCheckPermission() throws IOException, AccessControlException {
```

Comment:
`AccessControlException` is a subclass of IOE. I guess we don't need to declare it in the thrown list?

Reply:
ok ,I will amend it